components/_patterns/02-molecules/address-bar/address-bar.yml

```diff
@@ -0,0 +1,12 @@
+address_bar__date_start: '2119-11-15 12:00:00'
+address_bar__date_end: '2119-11-15 14:00:00'
```

Comment:
We write code that _lasts_!

(Jokes aside, I am seeing `2119` around and about a fair amount, so I assume this is intentional?)